tensorboard/plugins/hparams/tf_hparams_query_pane/tf-hparams-query-pane.ts

```diff
@@ -826,10 +836,15 @@ class TfHparamsQueryPane extends LegacyElementMixin(PolymerElement) {
     this._metrics.forEach((metric, index) => {
       let colParam = {
         metric: metric.info.name,
-        filterInterval: parseInputInterval(
+      } as any;
+      if (
+        metric.filter.interval.min.value !== '' ||
+        metric.filter.interval.max.value !== ''
```

Comment:
nit: I think readability would improve if you used a little helper isIntervalSet and used it here and in the hparams filter.

Reply:
Done.